packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/index.md

```diff
@@ -103,25 +86,25 @@ The values for a configuration object are:
 
 The configuration information in a shipping app is usually static. You can initialize the configuration values directly in the code or read them from a file. During development you may want to provide a way to edit configuration values.
 
-<StackSnippet snippet="loadingaconfiguration" />
+<StackSnippet snippet="initializeflow" />
 
-### Start sign-in
+### Start the sign-in
 
-Start the sign-in flow by creating an SDK client, and then requesting the first step.
+Start the sign-in flow after initializing an `InteractionCodeFlow` object.
 
 <StackSnippet snippet="initializingsdksession" />
 
 ### Process the response
 
-The steps for processing a response are:
+The steps for processing each response are:
 
-1. Check for a successful sign-in flow.
+1. Check if the sign-in succeeded.
```

Comment:
```suggestion
1. Check if the user signed in successfully.
```

Reply:
Agree it needs changing, but how about this:

> 1. Check for a successful sign-in.

Reply:
how about:
>Check for a successful sign-in flow.